### PR TITLE
emulator: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/emulator.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Display the help:
+
+`emulator -help`
+
+- Start an Android emulator device:
+
+`emulator -avd {{name}}`
+
+- Display the web cameras on your development computer that are available for emulation:
```
Comment (sbrl) [inline pages/common/emulator.md:14]: Perhaps this will read better?
```suggestion
- Display the webcams on your development computer that are available for emulation:
```
Not sure that we need to expand `webcam` there.

File: pages/common/emulator.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`emulator -avd {{name}} -webcam-list`
+
+- Start an emulator overriding the facing back camera setting (or -camera-front for front camera):
```
Comment (sbrl) [inline pages/common/emulator.md:18]: Perhaps this will read better?
```suggestion
- Start an emulator overriding the facing back camera setting (use `-camera-front` for front camera):
```

File: pages/common/emulator.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`emulator -avd {{name} -camera-back {{none|emulated|webcamN}}`
```
Comment (sbrl) [inline pages/common/emulator.md:20]:
```suggestion
`emulator -avd {{name}} -camera-back {{none|emulated|webcamN}}`
```

File: pages/common/emulator.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Start an emulator, with a maximun network speed:
+
+`emulator -avd {{name} -netspeed {{gsm|hscsd|gprs|edge|hsdpa|lte|evdo|full}}`
```
Comment (sbrl) [inline pages/common/emulator.md:24]: Perhaps this will read better?
```suggestion
`emulator -avd {{name}} -netspeed {{gsm|hscsd|gprs|edge|hsdpa|lte|evdo|full}}`
```

File: pages/common/emulator.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Start an emulator with network latency:
+
+`emulator -avd {{name}} -netdelay {{gsm|hscsd|gprs|edge|hsdpa|lte|evdo|none}}`
+
+- Start an emulator, making all TCP connections through a specified HTTP/HTTPS proxy:
+
+`emulator -avd {{name}} -http-proxy {{server}}:{{port}}`
```
Comment (sbrl) [inline pages/common/emulator.md:32]: How does it determine whether it's HTTP or HTTPS?

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the new page, @navarroaxel! I've left some comments below for you to review.

**Review by sbrl — APPROVED** [review]
Thanks, @navarroaxel!

File: pages/common/emulator.md (hunk 5 above)
Comment (navarroaxel) [reply pages/common/emulator.md:32]: This is the help page:
```
$ emulator -help-http-proxy
the Android emulator allows you to redirect all TCP connections through
a HTTP/HTTPS proxy. this can be enabled by using the '-http-proxy <proxy>'
option, or by defining the 'http_proxy' environment variable.

<proxy> can be one of the following:

  http://<server>:<port>
  http://<username>:<password>@<server>:<port>

the 'http://' prefix can be omitted. If '-http-proxy <proxy>' is not used,
the 'http_proxy' environment variable is looked up and any value matching
the <proxy> format will be used automatically
```
How should I format this?

Comment (sbrl) [reply pages/common/emulator.md:32]: Ah, I see. Perhaps a concrete example for `{{server}}:{{port}}` would help here? What about this?
```suggestion
- Start an emulator, making all TCP connections through a specified HTTP/HTTPS proxy (port number is required):

`emulator -avd {{name}} -http-proxy {{http://example.com:80}}`
```

Comment (navarroaxel) [reply pages/common/emulator.md:32]: hey @einverne, @schneiderl wdyt?

File: pages/common/adb-install.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# adb-install
+# adb install
```
Comment (mebeim) [inline pages/common/adb-install.md:1]: Is this right, should it be in a separate PR just for clarity?

Comment (navarroaxel) [reply pages/common/adb-install.md:1]: The last rule in https://github.com/tldr-pages/tldr/blob/master/contributing-guides/maintainers-guide.md says:
```
Although having push access allows committing directly to the repository, please create pull requests for all of your changes, except the simplest ones (e.g. typo fixes).
```
So, I think this could be committed to `master` without reviews.

Comment (mebeim) [reply pages/common/adb-install.md:1]: Sure, no problem for me, was just making sure it wasn't an oversight that you might have wanted to separate.

File: pages/common/emulator.md (hunk 5 above)
Comment (mebeim) [reply pages/common/emulator.md:32]: I agree with @sbrl here, seems clearer this way, we always prefer explicit examples if possible.

Comment (navarroaxel) [reply pages/common/emulator.md]: All suggestions applied, could you please review? @mebeim

**Review by mebeim — APPROVED** [review]
